libs/pages/clusters/src/lib/ui/page-clusters-create/step-general/step-general.tsx

```diff
@@ -80,8 +81,36 @@ export function StepGeneral(props: StepGeneralProps) {
             }}
             render={({ field }) => (
               <BlockContent title="Installation type">
+                <Popover.Root>
+                  <Popover.Trigger>
+                    <span className="text-sm cursor-pointer text-brand-500 hover:text-brand-600 transition font-medium mt-2">
+                      Which should I chose? <Icon className="text-xs" name={IconAwesomeEnum.CIRCLE_QUESTION} />
```

Comment:
not sur but it's `choose` no?

Reply:
Yes you're right it's a typo from the figma
https://github.com/Qovery/console/pull/1194/commits/69f36912034cfe937ef005bcf8be85eb9e73712b